### ISSUE TITLE
branch: a11yComboBox

### ISSUE THREAD

**Issue by jessegreenberg** [phetsims/molarity#43]
A one-off branch for accessible comboboxes, see phetsims/sun#333 and phetsims/sun#334.

**Comment by jessegreenberg** [comment]
I was supposed to use `grunt create-one-off --branch={{BRANCH}}` to make this branch, going to do that and try again.

**Comment by pixelzoom** [comment]
@jessegreenberg since ComboBox has been totally rewritten, does this branch still have any value?

**Comment by pixelzoom** [comment]
@jessegreenberg ping.

**Comment by jessegreenberg** [comment]
No, this can be deleted now.

**Comment by jessegreenberg** [comment]
It has been removed.

**Comment by zepumph** [comment]
Should be re-deleted as part of phetsims/special-ops#198

**Comment by zepumph** [comment]
Deleted
